src/pages/index.tsx

```diff
@@ -44,24 +39,21 @@ const Index: FC = () => {
       <Head title="Tasks" />
 
       <div className="container">
+        {!!error && <p>Something went wrong, please contact admin!</p>}
         {
-          (!!error) && (
-            <p>Something went wrong, please contact admin!</p>
-          )
-        }
-        {
-          (isLoading)
+          isLoading
             ? (
               <p>Loading...</p>
-            )
-            : (
+            ) : (
               <div className={classNames.container}>
-                <Accordion title="Active">
-                  {renderCardList(activeTasks)}
-                </Accordion>
-                <Accordion title="Completed">
-                  {renderCardList(completeTasks)}
-                </Accordion>
+                {
+                  Object.keys(filteredTask).length > 0
+                    ? Object.keys(filteredTask).map((key) => (
+                      <Accordion title={key}>
```

Comment:
`key` is missing here, can we please add it.

Reply:
Do we really need to add it?
As ESLINT is not having issues also we are having a unique title prop attached to Accordion.

Reply:
https://reactjs.org/docs/lists-and-keys.html

Reply:
> https://reactjs.org/docs/lists-and-keys.html
> @ice-rahul You might want to go through this.

thanks for the reference 👍